src/mongo/db/fts/fts_language.cpp

```diff
@@ -28,6 +28,8 @@
  *    it in the license file.
  */
 
+#define MONGO_LOG_DEFAULT_COMPONENT ::mongo::logger::LogComponent::kDefault
```

Comment:
log functionality is not used, so no need to add this.
(I guess it was used for debugging and then debug code was removed)